Vizzuality/waterbird-population
# Feature/results detail table

This PR adds three new components:

 - Population trends - results detail in explore section
 - Tooltip component 
 - Comments component - to be rendered by tooltip when data has comment to show 

New style-sheet for component button for consistency trough project
Data model in population trend pending to be revised (waiting for real data)

## Testing instructions

https://waterbird-population-ibm355sfq.now.sh/explore
Scroll down to 'Population trend' section 
   -> check expand/collapse
   -> open comments 

## Pivotal Tracker

https://www.pivotaltracker.com/story/show/171882832
